src/odemis/driver/hamamatsurx.py

```diff
@@ -1093,6 +1135,7 @@ def readCommandResponse(self):
                         continue  # return to try-statement and start receiving again
 
                     if self._getReadoutCamInfo:
+                        # TODO leave as it is? and remove commented code?
```

Comment:
If the commented code doesn't work, just remove it (but leave the comment about the big hack here).